### DISCUSSION TITLE
Day 1 Retrospective - Please answer the following questions.

### DISCUSSION THREAD

**Discussion by davidgitit** [Drook-tech/Bootcamp#7] [Q&A]
What do we do well?

What could we have done better?

Ideas and suggestions?

**Comment by aviijtdas** [comment]
What do we do well? --> Good coverage of topics, especially demo
What could we have done better? NA
Ideas and suggestions? NA

**Comment by Saraquin** [comment]
Done well:
I found the run through of the portal live very useful

Do better:
more reference to material on some topics as they are touched on so we can review further on areas of interest.

Recommendations:
Single dedicated question method?
timeline overlay on the video stream so we know what subject is being talked on?

**Comment by william-liebenberg** [comment]
What do we do well?

- Good overview of GH features

What could we have done better?

- N/A :)

Ideas and suggestions?

- Demo Monkey - one of the participants to share their screen and GH Presenter gives them instructions to demonstrate one of the features (e.g. setting up Projects Beta and adding issues)

**Comment by wicksipedia** [comment]
> What do we do well?

- Covered a lot of information in a 3h session
- Dogfooding - using discussions as part of the bootcamp

> What could we have done better?

- n/a - given the high level nature of the day

> Ideas and suggestions?

- Collect some of the resources/materials together into a repo e.g. awesome-github-bootcamp-info

**Comment by jayesh-talreja** [comment]
overall session was very informative and would like to see more hands on demo in actions part tomorrow

**Comment by ramesh2051** [comment]
What do we do well? 
Covered a lot of information topics, especially demo in Wiki, issues and everything
What could we have done better? 
N/A
Ideas and suggestions? 
N/A
